example/spec/emsReport/emsReportSpec.js

```diff
@@ -0,0 +1,71 @@
+const moment = require('moment');
+const { Lambda } = require('aws-sdk');
+const { loadConfig } = require('../helpers/testUtils');
+const aws = require('@cumulus/common/aws');
+
+const config = loadConfig();
+
+const emsReportLambda = `${config.stackName}-EmsReport`;
+const bucket = config.bucket;
+const emsProvider = config.ems_provider;
+const stackName = config.stackName;
+
+describe('The EMS report', () => {
+  describe('The EmsReport lambda is invoked daily', () => {
+    let expectReports = false;
+    beforeAll(async () => {
+      const lambda = new Lambda();
+      const lambdaConfig = await lambda.getFunctionConfiguration({ FunctionName: emsReportLambda })
+        .promise();
+      const lastUpdate = lambdaConfig.LastModified;
+
+      // Compare lambda function's lastUpate with the time 24 hours before now.
+      // If the lambda is created 24 hours ago, it must have been invoked
+      // and generated EMS reports for the previous day.
+      if (new Date(lastUpdate).getTime() < moment.utc().subtract(24, 'hours').toDate().getTime()) {
+        expectReports = true;
+      }
+    });
+
+    it('reports are generated', async () => {
+      if (expectReports) {
+        const datestring = moment.utc().format('YYYYMMDD');
+        const types = ['Ing', 'Arch', 'ArchDel'];
+        const jobs = types.map((type) => {
+          const filename = `${datestring}_${emsProvider}_${type}_${stackName}.flt`;
+          const key = `${stackName}/ems/${filename}`;
+          return aws.fileExists(bucket, key);
+        });
+        const results = await Promise.all(jobs);
+        results.forEach((result) => expect(result).not.toBe('false'));
+      }
+    });
+  });
+
+  describe('The EmsReport lambda can be run at any time', () => {
+    let lambdaOutput;
+    beforeAll(async () => {
+      const lambda = new Lambda();
+      const response = await lambda.invoke({ FunctionName: emsReportLambda }).promise();
```

Comment:
This seems like cheating for this test case.  Operators won't invoke the lambda directly, they'll need to do it through the API or dashboard.

Reply:
Actually, reading through the code, it looks like operators can't invoke it via API, so maybe I'm missing the test case here and it's just a wording thing.  It may be sufficient to rephrase the `describe` to "After execution"